desktop/src/main.rs

```diff
@@ -107,7 +99,43 @@ fn run() -> Result<(), Error> {
                 Action::Nothing => {}
             }
         }
+
+        // Write FPS into window title
+        if let Some(fps) = loop_helper.report_rate() {
+            window.set_title_postfix(&format!("{:.1} FPS", fps));
+        }
+
+        // Sleep for a while to reach our target FPS
+        if !window.in_turbo_mode() {
```

Comment:
I think we should limit the speed even if we are in turbo mode. Maybe just double the fps limit in turbo mode?